README.md

```diff
@@ -1,152 +1,131 @@
-   ```
-
-   _(Use `-b` to combine build and run steps.)_
-
-5. **Connect to the Web Interface**:
-
-   Open the web interface at [http://localhost:5001](http://localhost:5001) or with the port specified during `init`.
-
-6. **Send a REST Request**:
-
-   Try the system by sending a request to the REST API gateway interface.
-
-   ```sh
-   curl --location 'http://localhost:5050/api/v1/request' \
-   --header 'Authorization: Bearer None' \
-   --form 'prompt="What is the capital of France?"' \
-   --form 'stream="false"'
-   ```
-
-Learn about [Agents](https://solacelabs.github.io/solace-agent-mesh/docs/documentation/concepts/agents) and [Gateways](https://solacelabs.github.io/solace-agent-mesh/docs/documentation/concepts/gateways) to add more functionalities to your project.
-
-For full details, see the [Quick Start Guide](https://solacelabs.github.io/solace-agent-mesh/docs/documentation/getting-started/quick-start).
+#### Once running:
 
-## Why Use Solace Agent Mesh?
+- Open the Web UI at [http://localhost:5001](http://localhost:5001) to talk with a chat interface.
+<details>
+  <summary>Use the REST API directly via curl</summary>
 
-Building production-ready AI applications presents unique challenges. While it's relatively easy to create AI prototypes, deploying them in enterprise environments requires solving complex problems around integration, scalability, and observability. The Solace Agent Mesh addresses these challenges through:
+  ```bash
+  curl --location 'http://127.0.0.1:5050/api/v1/request' \
+    --form 'prompt="What is the capital of France?"' \
+    --form 'stream="false"'
+  ```
 
-- **Composable Architecture**: Start with a small number of agents and gateways. You can add more over time.
+</details>
 
-- **Add Agents to Increase Capabilities**: Each new agent adds more capabilities to the system. Adding a new agent isn't additive - it is exponential. With each agent being able to enhance all other agents as they collaborate for more and more complex tasks.
 
-- **Add Gateways to Increase Use Cases**: Each new gateway opens up new use cases for the system. A new gateway can provide a new interface to the system, with a different system purpose and response rules.
-- **Event-Driven Design**: Built on proven event-driven architecture principles, providing complete decoupling of components. This makes the system highly flexible and resilient, while enabling real-time monitoring of every interaction.
+---
 
-- **Enterprise-Ready**: Designed from the ground up for production deployments that incorporates the experience from Solace in building mission-critical distributed systems.
+## ➡️ Next Steps
 
-## Next Steps
+Want to go further? Here are some hands-on tutorials to help you get started:
 
-Check [Solace Agent Mesh Documentation](https://solacelabs.github.io/solace-agent-mesh/docs/documentation/getting-started/introduction) to learn more about the Solace Agent Mesh.
+| 🔧 Integration | ⏱️ Est. Time | 📘 Tutorial |
```

Comment:
For my taste, I think this is a bit too emoji laden. While it isn't critical right now, I think perhaps a 50% reduction in text leading emojis (these headings would be a place to start). I do think they liven up the page, but if used too much I think it can take away a bit from this being an enterprise grade solution (speaking as an old guy here...)